action.yml

```
@@ -377,6 +377,8 @@ runs:
               command.append('--continue-on-error')
           if gha_integration_diff:
               command.append('--diff')
+
+      if gha_testing_type == 'integration' or gha_testing_type == 'unit':
```

I'd probably write this as:
```suggestion
      if gha_testing_type in {'integration', 'unit'}:
```